package.json

```diff
@@ -92,9 +93,7 @@
     "build-dist": "bash ./scripts/build-dist.sh",
     "build-win-installer": "scripts\\build-windows-installer.bat",
     "check-lockfile": "./scripts/check-lockfile.sh",
-    "lint": "yarn run lint-prettier && eslint . && flow check",
-    "lint-prettier": "node scripts/prettier.js lint",
-    "prettier": "node scripts/prettier.js write",
```

Comment:
So has the script been deleted? I don't see it in the changes. And if you're removing the write script, then prettier will no longer auto-magically format the code and only warn. Is that the intention? If not, I'd add a script like `lint-fix` and pass in the `eslint --fix` command.